scripts/build_third_party_licenses.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A script to generate a THIRD_PARTY_LICENSES file containing all the licenses that we use from third parties.
+# This is intended to be run from the project root as `$ ./scripts/build_third_party_licenses.sh`
+# NOTE: This only adds licenses from go dependencies. For other licenses, see NOTICE.md
+
+# The apache 2.0 license doesn't get modified with a copywrite. To reduce duplication, add attribution for each project using the license, but include the license text just once.
+go-licenses report --template=scripts/third_party_licenses/apache.tpl --ignore github.com/awslabs/soci ./... > ./THIRD_PARTY_LICENSES
+cat ./scripts/third_party_licenses/APACHE_LICENSE >> ./THIRD_PARTY_LICENSES
```

Comment:
Similarly, this assumes the script is running at SOCI root. It should be changed to `cat ${SOCI_SNAPSHOTTER_PROJECT_ROOT}/scripts/... >> ${SOCI_SNAPSHOTTER_PROJECT_ROOT }/THIRD_PARTY_LICENSES`